.circleci/config.yml

```diff
@@ -65,21 +65,39 @@ commands:
           paths:
             - ./node_modules
 
-  app-build-and-store:
```

Comment:
`app-build-and-store` command is gone to reduce code clutter. This command is used only by `app-build-and-store-job` job. Steps are combined into `app-build-and-store-job` job.